source/manual/receive-publisher-emails-from-email-alert-api-in-integration-and-staging.html.md

```diff
@@ -0,0 +1,15 @@
+---
+owner_slack: "#govuk-developers"
+title: Receive publisher emails from Email Alert API in integration and staging
```

Comment:
I'm not sure if this makes sense because the publishing apps don't go through the Email Alert API to send emails, they use Notify directly.